DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgWordRatioRepository.cs

```diff
@@ -48,6 +41,37 @@ public int Add(WordRatioModel entity)
             });
     }
 
+    public int AddWordRatios(IEnumerable<WordRatioModel> entities)
+    {
+        int rowsAffected = 0;
+        IEnumerable<WordRatioModel> wordRatios = entities.ToList();
+        using IDbConnection con = _connectionFactory.CreateConnection();
+        IDbTransaction transaction = con.BeginTransaction();
+        foreach (WordRatioModel entity in wordRatios)
+        {
+            rowsAffected += con.Execute(
+                "insert into word_ratios(documents_id, word, amount, percent, rank)" +
+                " values (@DocumentId, @Word, @Amount, @Percent, @Rank)",
+                new
+                {
+                    entity.DocumentId,
+                    entity.Word,
+                    entity.Amount,
+                    entity.Percent,
+                    entity.Rank
+                }, transaction: transaction);
+        }
+
+        if (rowsAffected != wordRatios.Count())
+        {
+            transaction.Rollback();
+            throw new RowsAffectedMismatchException();
+        }
+
+        transaction.Commit();
+        return rowsAffected;
+    }
```

Comment:
Very nice - for ren performance kunne man skrive det her om til at bygge én stor query ligesom det Dam har præsenteret, så er vi også consistent 😳

Reply:
@damniko Vil du være en darling og implementere et rework med dine nye gyldne drops? 👀